spyder/utils/programs.py

```diff
@@ -462,6 +462,29 @@ def is_module_installed(module_name, version=None, installed_version=None,
             
             return check_version(actver, version, symb)
 
+def is_python_interpreter_valid_name(filename):

```

Comment:
Please add a test to verify this function is working correctly (as @goanpeca asked you)